hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -811,6 +863,32 @@ func reconcileUserDataSecret(userDataSecret *corev1.Secret, nodePool *hyperv1.No
 	return nil
 }
 
+func reconcileTunedConfigMap(tunedConfigMap *corev1.ConfigMap, nodePool *hyperv1.NodePool, tunedConfig string) error {
+	tunedConfigMap.Immutable = k8sutilspointer.BoolPtr(false)
+	if tunedConfigMap.Annotations == nil {
+		tunedConfigMap.Annotations = make(map[string]string)
+	}
+	if tunedConfigMap.Labels == nil {
+		tunedConfigMap.Labels = make(map[string]string)
+	}
+
+	tunedConfigMap.Labels[tunedConfigMapLabel] = "true"
+	tunedConfigMap.Annotations[nodePoolAnnotation] = client.ObjectKeyFromObject(nodePool).String()
+
+	if tunedConfigMap.Data == nil {
```

Comment:
it might be simpler to do:
```go

	if tunedConfigMap.Data == nil {
		tunedConfigMap.Data = map[string]string{}
        }
	tunedConfigMap.Data[tunedConfigKey] = tunedConfig
```

Reply:
Certainly... Thanks, this should be fixed.